libnavigation-metrics/src/main/java/com/mapbox/navigation/metrics/extensions/MetricEx.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.navigation.metrics.extensions
+
+import com.mapbox.android.telemetry.Event
+import com.mapbox.navigation.base.metrics.DirectionsMetrics
+import com.mapbox.navigation.base.metrics.MetricEvent
+import com.mapbox.navigation.base.metrics.NavigationMetrics
+
+fun MetricEvent.toTelemetryEvent(): Event? =
+    when (metric) {
+        DirectionsMetrics.ROUTE_RETRIEVAL,
+        NavigationMetrics.ARRIVE,
+        NavigationMetrics.CANCEL_SESSION,
+        NavigationMetrics.DEPART,
+        NavigationMetrics.REROUTE,
+        NavigationMetrics.FEEDBACK,
+        NavigationMetrics.INITIAL_GPS,
+        NavigationMetrics.APP_USER_TURNSTILE -> this as Event
+        else -> null
+    }
```

Comment:
We have `libnavigationutils` where we have defined extensions. Can this be moved to that library?

Reply:
In general case - yes, of course, we move all `utils` and `extensions` to `libnavigation-utils` module. But this extension depends on `telemetry` library and `metrics` package constants (`NavigationMetrics.ARRIVE` and so on). I'm not sure is it bad when `utils` depends on `metrics` package, but I think it is 100% not good if we make `utils` module depending on `telemetry` external library.